proto/grp.proto

```diff
@@ -17,4 +20,45 @@ message Group {
 
   // True if this group has linked a Github token.
   bool github_linked = 4;
+
+  // Whether users from the owned domain should automatically
+  // be added to the group when they sign up (defaults to false).
+  //
+  // Not yet implemented.
```

Comment:
See below re CreateGroupRequest but I think you can remove this.

Reply:
You mean remove `auto_populate_from_owned_domain`? I thought we wanted that functionality to be optional?

Reply:
Ah you mean this doesn't need to be in the Group object. Removed

Reply:
yup, you got it! sorry I wasn't clear.